app/Models/UserCredit.php

```diff
@@ -37,11 +37,11 @@ class UserCredit extends Model
 
     protected $guarded = [];
 
-    public function type(){
+    public function type() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->